core/src/main/java/com/google/instrumentation/trace/Tracer.java

```diff
@@ -31,14 +31,21 @@
  * <p>Users may choose to use manual or automatic Context propagation. Because of that this class
  * offers APIs to facilitate both usages.
  *
+ * <p>The automatic context propagation is done using {@link io.grpc.Context} which is a gRPC
```

Comment:
We are working on this. Please file an issue if you are concerned about this because it may help use to prioritize the work.